website/src/index.html

```diff
@@ -424,17 +346,17 @@ <h1>SUBMIT A SOLUTION</h1>
             <h1>FAQ</h1>
             <p>
               <b>
-                Where can I ask questions?
+                Q. Where can I ask more questions?
               </b><br />
-              Check out our <a href="https://groups.google.com/g/climate-cooperation-competition" target="blank">Google
+                Check out our <a href="https://groups.google.com/g/climate-cooperation-competition" target="blank">Google
                 Group</a> and <a
                 href="https://join.slack.com/t/aieconomist/shared_invite/zt-g71ajic7-XaMygwNIup~CCzaR1T0wgA"
                 target="blank">Slack
                 channel</a>.
             </p>
             <p>
               <b>
-                How can I find other people interested in the competition?
+                Q. How can I find other people interested in the competition?
```

Comment:
Don't need to add a "Q. ". It's obvious from bolding and the question mark it's a question.